### DISCUSSION TITLE
Add DOI creation tool to Download Specimen Results

### DISCUSSION THREAD

**Discussion by campmlc** [ArctosDB/arctos#4968] [General]
Dealing with Attribution issues - we need people who are using our data to be able to cite data downloads easily, with one click. Rather than go through Archive Search, I would like to have a means to instantly create a DOI for a downloaded data set from the Download screen, and provide that to the researcher in the download. Is this possible? Using GBIF as a model?

**Comment by jrdemboski** [comment]
This is coming out of BCoN discussions

**Comment by campmlc** [comment]
We could also have a way to link the DOIs created from any data downloads to the specimen records themselves. So if someone downloads data and creates a DOI, then later cites that DOI in a paper as a used dataset, the actual specimen records would be linked to that dataset/publication via the DOI. Or, it generates an email to the collection managers who can approve or not, and that would provide notification that these data are being used.

**Comment by campmlc** [comment]
There is discussion now about this being done through GBIF and iDigBio - we need it to happen on all levels.

**Comment by dustymc** [comment]
See also https://github.com/ArctosDB/arctos/issues/1130

Providing a citable dataset requires a curatorial commitment. Locked Archives do that. They're only available to manage_collection users - the only folks who can make that commitment - and they prevent deleting, encumbering, etc. specimens. They can have DOIs.

There are some limitations on the number of DOIs we can get (under our current arrangement), and creating them invokes a commitment as well. It would be technically trivial so I'm happy to discuss, but I suspect that maintaining the sort of "research grade" data that DOIs implicitly promise forever (which is the point of DOIs) for user-defined "nodes" would be very difficult (eg, what happens when we change the structure of something they used in their search, or when you delete a specimen included in that search?)

Users wanting an archive of specimen data will need to arrange that themselves. We can facilitate of course, but I have no way of knowing what data they're concerned with etc. GBIF (and everything else dealing in DWC data) has very limited data which they get it from us in a flat format, so they're not really a great comparison in that regard.

**Comment by campmlc** [comment]
Example GBIF download - this is still too much info in a confusing format - we need a big "HERE IS YOUR DOI - PLEASE CITE IT" box.


Hello mlcampbell,

Your download is available at the following address:
    http://api.gbif.org/v1/occurrence/download/request/0005678-180131172636756.zip

When using this dataset please use the following citation:
    GBIF.org (14th February 2018) GBIF Occurrence Download https://doi.org/10.15468/dl.hcus83

Download Information:
    DOI: https://doi.org/10.15468/dl.hcus83 (may take some hours before being active)
    Creation Date: Wednesday, 14 February 2018 21:06:36 o'clock GMT
    Records included: 80 records from 8 published datasets
    Data size: 0 B
    Download format: SIMPLE_CSV
    Filter used: TaxonKey: Corrodopsylla curvata (Rothschild, 1915)

Download file retention:
    Information about this download will always be available at https://doi.org/10.15468/dl.hcus83 and https://www.gbif.org/occurrence/download/0005678-180131172636756

    The SIMPLE_CSV ZIP file will be kept for six months (until 14th August 2018).  You can ask
    us to keep the file for longer from https://www.gbif.org/occurrence/download/0005678-180131172636756

    If you cite this download using the DOI, we will usually detect this and keep the file indefinitely.

    For more information on this, see https://www.gbif.org/faq/?question=for-how-long-will-does-gbif-store-downloads

For help with opening downloaded files, see
    https://www.gbif.org/faq?question=opening-gbif-csv-in-excel
or the FAQ section of the GBIF website:
    https://www.gbif.org/faq

**Comment by dustymc** [comment]
I like that the GBIF Comma Separated Values (CSV) file is separated by tabs....

```HERE IS YOUR DOI - PLEASE CITE IT```

>what happens when we change the structure of something they used in their search, or when you delete a specimen included in that search?

I think "research grade" requires more than applying what's normally a permanent identifier to an ephemeral dataset, or explaining that somewhere, or SOMETHING of the sort.

> the actual specimen records would be linked to that dataset/publication via the DOI.

That is (sorta) supported, but it'll just take them to a semi-random record. http://www.doi.org/multiple-url-resolution.html

>generates an email to the collection managers who can approve or not, 

Once a DOI is assigned, it is/should be permanent. We could do something like this with Locked Archives, somehow, I think, maybe. I think a Curator committing to preserve the dataset (eg, by locking an Archive) is a critical part of forever-citations. I certainly like the idea of streamlining that process, I just don't know how to implement it.

Should we put this back on the AWG agenda?

**Comment by campmlc** [comment]
Yes, please, let's discuss.

**Comment by mkoo** [comment]
This needs a fresh consideration. A lot things have changed in last year except for one, the nontrivial infrastructure and commitment to DOI minting to do this properly.

I'm willing to move this to Needs Discussion
